go/consensus/tendermint/api/api.go

```diff
@@ -117,7 +126,12 @@ func (bld *EventBuilder) Attribute(key, value []byte) *EventBuilder {
 // The typed attribute is automatically converted to a key/value pair where its EventKind is used
 // as the key and a CBOR-marshalled value is used as value.
 func (bld *EventBuilder) TypedAttribute(value TypedAttribute) *EventBuilder {
```

Comment:
This method could automatically try to determine whether the passed attribute is also a `CustomTypedAttribute` to avoid an additional method.